packages/extension/src/ui/features/actions/ActionScreen.tsx

```diff
@@ -32,7 +32,7 @@ export const ActionScreen: FC = () => {
   const [action] = actions
   const isLastAction = actions.length === 1
   const signerAccount = useAccount(action.type === 'ALPH_TRANSACTION' && selectedAccount
-    ? { address: action.payload.params.signerAddress, networkId: action.payload.params.networkId ?? selectedAccount.networkId }
+    ? { address: action.payload[0].params.signerAddress, networkId: action.payload[0].params.networkId ?? selectedAccount.networkId }  // FIXME: always use the first payload
```

Comment:
Do we need to fix this? It seems like it doesn't matter since we always sign the transaction with the specified address in `TransactionParams`.